src/components/JoinUs/elements.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+import styled from 'styled-components'
+import { Padding } from 'styled-components-spacing'
+import breakpoint from 'styled-components-breakpoint'
+import { Grid, Row, Col } from '../grid'
+import { H3, Paragraph, H5 } from '../Typography'
+import GreyBG from '../GreyBG'
+import Hr from '../Common/Hr'
+
+/**
+ * Layout
+ */
+export const FirstColumn = ({ children }) => (
+  <Col width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>{children}</Col>
+)
+
+export const ShiftedColumn = styled(Col)`
+  ${breakpoint('tablet')`
+    margin-left: 8.33333%;
+  `};
+`
+export const SecondColumn = ({ children }) => (
+  <ShiftedColumn width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>{children}</ShiftedColumn>
+)
+
+export const List = ({ list }) => (
+  <Padding top={1}>
+    <ReactMarkdown
+      renderers={{
+        heading: props => <H5 as="h4" noMargin bold {...props} />, // eslint-disable-line react/display-name
+        paragraph: props => <Paragraph muted {...props} /> // eslint-disable-line react/display-name
+      }}
+      source={list}
+    />
+  </Padding>
+)
+
+export const Section = ({ children, greyBg, ...props }) => {
+  const content = (
+    <Grid {...props}>
+      <Padding
+        top={{ smallPhone: 3, tablet: 4 }}
```

Comment:
Asked Tom in person, what this rule is supposed to be. I am told:
- desktop & large tablet - top: 24, bottom: 36, left: 24, right: 24
- small tablet: top: 18, bottom: 24, left: 18, right: 18
- mobile: top: 24, bottom: 24, left: 24, right: 24

Note: we're only gonna be able to change this once [the typography components get sorted out](https://trello.com/c/PRcbDKOg/247-please-align-typographic-padding-values-with-the-design-documentation-including-top-and-bottom-padding)

Reply:
So, is there any action to be taken regarding this file, for the time being?

Reply:
Depends whether designers are happy pushing this page live before the typography components are fixed

Reply:
I'm guessing that this `Section` component does not depend on the typography, right? It's just an aux component to replicate the paddings set on the designs? Or am I looking at the wrong thing?